app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -16,12 +16,11 @@
     </div>
     <div class="form-wrapper">
       <v-alert
-        :color="saving ? 'primary' : undefined"
-        dense
-        text
-        transition="scale-transition"
-        :type="saving ? 'info' : 'success'"
         :value="saved || saving"
+        :class="saving ? NOTIFICATIONS_TYPES.INFO.class : NOTIFICATIONS_TYPES.SUCCESS.class"
+        :color="saving ? NOTIFICATIONS_TYPES.INFO.color : NOTIFICATIONS_TYPES.SUCCESS.color"
+        :icon="saving ? NOTIFICATIONS_TYPES.INFO.icon : NOTIFICATIONS_TYPES.SUCCESS.icon"
```

Comment:
There may be situations where saving fails due to network traffic issues or other random acts of chaos. This only handles the successful case and implies success even when something could have failed.

Reply:
https://github.com/bcgov/common-hosted-form-service/pull/212/files#r671432532